package.json

```diff
@@ -80,6 +83,13 @@
     "@types/storybook__addon-actions": "^3.4.2",
     "@typescript-eslint/eslint-plugin": "1.13.0",
     "@typescript-eslint/parser": "1.13.0",
+    "@wdio/cli": "5.11.14",
```

Comment:
could theses be subdependencies of instantsearch-e2e-tests#v1.1.0 ?

Reply:
They could be, but I choose to put them as `peerDependencies` since the `wdio` command will be run from `instantsearch.js`. Also it gives us more control having them as top dependencies.